src/main/java/org/opensearch/security/configuration/ConfigurationRepository.java

```diff
@@ -136,10 +136,11 @@ public void run() {
                                     ConfigHelper.uploadFile(client, cd+"roles_mapping.yml", securityIndex, CType.ROLESMAPPING, DEFAULT_CONFIG_VERSION);
                                     ConfigHelper.uploadFile(client, cd+"internal_users.yml", securityIndex, CType.INTERNALUSERS, DEFAULT_CONFIG_VERSION);
                                     ConfigHelper.uploadFile(client, cd+"action_groups.yml", securityIndex, CType.ACTIONGROUPS, DEFAULT_CONFIG_VERSION);
+                                    final boolean populateEmptyIfFileMissing = true;
```

Comment:
Should this be moved to a config constants file?

Reply:
Sure will do in next commit. Thanks for the suggestion.

Reply:
It was already always true. We just moved the line of declaring this variable. From next commit onwards, we can change this from config constants file.